rkt/status.go

```diff
@@ -95,6 +95,16 @@ func printStatus(p *pod) error {
 		stdout.Printf("started=%s", startedStr)
 	}
 
+	finished, err := p.getFinishTime()
+	if err != nil {
+		return errwrap.Wrap(fmt.Errorf("unable to get finish time for pod %q", p.uuid), err)
+	}
+	var finishedStr string
```

Comment:
I know you're following the code above but there's no need to declare `finishedStr` here, you can just write below:

```go
stdout.Printf("finished=%s", finished.Format(defaultTimeLayout))
```

The same with `started=%s`.